### PR TITLE
Added Spicetify

### PR DESCRIPTION
**Pull request by curiousvlxd**
## Project URL
https://github.com/spicetify/spicetify-cli

## Category
Music

## Description
I added Spicetify to the `application.json`
 
## Checklist
- [X] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [X] Only one project/change is in this pull request
- [X] Screenshots(s) added if any
- [X] Has a commit from less than 2 years ago
- [X] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -4315,6 +4315,22 @@
...
+        {
+            "short_description": "Command-line tool to customize the official Spotify client. Supports Windows, MacOS and Linux.",
+            "categories": [
+                "music"
+            ],
+            "repo_url": "https://github.com/spicetify/spicetify-cli",
+            "title": "spicetify-cli",
+            "icon_url": "",
+            "screenshots": [
+                "https://github.com/spicetify/spicetify-cli/blob/master/.github/assets/logo.png"
```
Comment (serhii-londar) [inline applications.json:4327]: @curiousvlxd Please use link to an image - https://raw.githubusercontent.com/spicetify/spicetify-cli/master/.github/assets/logo.png

**Review by serhii-londar — CHANGES_REQUESTED** [review]

File: applications.json (hunk 1 above)
Comment (curiousvlxd) [reply applications.json]: > @curiousvlxd Please use link to an image - https://raw.githubusercontent.com/spicetify/spicetify-cli/master/.github/assets/logo.png

Done! [commit link](https://github.com/serhii-londar/open-source-mac-os-apps/pull/816/commits/9295b40de2506bed9f1f9f724d357491372d7994)

**Review by curiousvlxd — COMMENTED** [review]
I already changed it, but I will sumbit your request too.

**Comment by serhii-londar** [comment]
@curiousvlxd Thanks! 🚀

**Pull request merged**